### PR TITLE
libsigcpp: conan v2 support

### PR DESCRIPTION
**Pull request by SpaceIm**
Specify library name and version:  **lib/1.0**

---

- [ ] I've read the [contributing guidelines](https://github.com/conan-io/conan-center-index/blob/master/CONTRIBUTING.md).
- [ ] I've followed the [PEP8](https://www.python.org/dev/peps/pep-0008/) style guides for Python code in the recipes.
- [ ] I've used the [latest](https://github.com/conan-io/conan/releases/latest) Conan client version.
- [ ] I've tried at least one configuration locally with the [conan-center hook](https://github.com/conan-io/hooks.git) activated.

### CODE REVIEW

**Review by uilianries — DISMISSED** [review]
LGTM

File: recipes/libsigcpp/2.x.x/conanfile.py [hunk 1]
```diff
@@ -1,20 +1,23 @@
...
-required_conan_version = ">=1.43.0"
+required_conan_version = ">=1.53.0"
 
 
-class LibSigCppConanV2(ConanFile):
+class LibSigCppConan(ConanFile):
     name = "libsigcpp"
     homepage = "https://github.com/libsigcplusplus/libsigcplusplus"
     url = "https://github.com/conan-io/conan-center-index"
     license = "LGPL-3.0"
```
Comment (jwillikers) [inline recipes/libsigcpp/2.x.x/conanfile.py:18]: Could you update the license to be either `LGPL-3.0-only` or `LGPL-3.0-or-later` here? The `LGPL-3.0` SPDX identifier is deprecated.

File: recipes/libsigcpp/3.x.x/conanfile.py [hunk 2]
```diff
@@ -14,7 +19,7 @@ class LibSigCppConan(ConanFile):
     url = "https://github.com/conan-io/conan-center-index"
     license = "LGPL-3.0"
```
Comment (jwillikers) [inline recipes/libsigcpp/3.x.x/conanfile.py:20]: Same thing about the license here.

**Review by jwillikers — CHANGES_REQUESTED** [review]

File: recipes/libsigcpp/2.x.x/conanfile.py (hunk 1 above)
Comment (SpaceIm) [reply recipes/libsigcpp/2.x.x/conanfile.py:18]: LGPL-3.0-only seems to be the appropriate equivalent

**Review by jwillikers — APPROVED** [review]
Thank you!

**Review by uilianries — APPROVED** [review]
LGTM

**Pull request merged**